src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/EmbeddingProvidersConfig.java

```diff
@@ -220,6 +220,10 @@ interface RequestProperties {
 
       @Nullable
       Optional<String> taskTypeRead();
+
+      /** Batch size supported by the provider. */
+      @WithDefault("2048")
```

Comment:
Do we want to use defaults in this case? I'd guess every provider must have their specific maximum; many have lower than 2048.